bindings/node/lib/autoEncrypter.js

```diff
@@ -159,11 +159,18 @@ module.exports = function (modules) {
       // that we are not using the CSFLE shared library.
       if (!this._bypassMongocryptdAndCryptShared && !this.cryptSharedLibVersionInfo) {
         this._mongocryptdManager = new MongocryptdManager(options.extraOptions);
-        this._mongocryptdClient = new MongoClient(this._mongocryptdManager.uri, {
+        const clientOptions = {
           useNewUrlParser: true,
           useUnifiedTopology: true,
           serverSelectionTimeoutMS: 10000
-        });
+        };
+
+        const wasURIProvided = typeof options.extraOptions === 'object' && !!options.extraOptions.mongocryptdURI;
```

Comment:
I think you need a null check here before accessing the `mongocryptdURI` key

Reply:
That's out of the bounds of our types, both in the jsdoc types and Typescript.  Is this something you still think we should guard against?

Reply:
Yup, but our type here is explicitly "string", so null wouldn't be allowed.  This gets to a larger question about whether we should test outside the bounds of what our types declare that we support.  

I updated the check to match the other checks in the file though